src/templates/domain/useCases/create.ejs

```diff
@@ -22,17 +23,16 @@ const create<%- props.name.pascalCase %> = injection =>
     //Step description and function
     'Check if the <%- props.name.raw %> is valid': step(ctx => {
       ctx.<%- props.name.camelCase %> = <%- props.name.pascalCase %>.fromJSON(ctx.req)
-      <% if(!props.mongo){ %>ctx.<%- props.name.camelCase %>.id = Math.floor(Math.random() * 100000).toString()<% } %>
-      
-      if (!ctx.<%- props.name.camelCase %>.isValid()) 
+      ctx.<%- props.name.camelCase %>.id = uuid.v4()
```

Comment:
it seems a mongo `if` was removed. is it working with mongo?

Reply:
Yes. Worked perfectly